README.md

```diff
@@ -179,3 +191,55 @@ Until we make changes, you'll need to have at least one subscription before you
      user_expressions: {},
      payload: [] } }
 ```
+
+## Contributors and developers
+
+### ZeroMQ Dependency
+
+If you plan to contribute to this project or extend it, you will need
+to have ZeroMQ installed on your system.
+
```

Comment:
Oh! We rely on `zmq-prebuilt` here now so technically they don't have to go through all this yucky setup anymore.